conda-recipe/run_test.sh

```diff
@@ -48,28 +48,34 @@ if [[ "$*" == *"--json-report"* ]]; then
         rm .pytest_reports/*.json
     fi
 fi
-function json_report_name {
+
+function generate_pytest_args {
+    declare -a ARGS=()
     if [[ "${with_json_report}" == "1" ]]; then
-        printf -- "--json-report-file=.pytest_reports/$1_report.json"
+        ARGS+=("--json-report-file=.pytest_reports/$1_report.json")
+    fi
+    if [ -n "${COVERAGE_RCFILE}" ]; then
+        ARGS+=(--cov=onedal --cov=sklearnex --cov-config="${COVERAGE_RCFILE}" --cov-append --cov-report=)
     fi
+    "${ARGS[@]}"
```

Comment:
I think this doesn't do the same thing. It would end up trying to execute a command instead of adding arguments to an existing call.

Reply:
good catch, we should schedule a meeting or make a chat to iron out these issues quickly. i have no particular preference